_inc/lib/admin-pages/class-jetpack-about-page.php

```diff
@@ -542,7 +554,42 @@ public function display_plugins() {
 	}
 
 	/**
-	 * Fetch Gravatar hashes for public A12s from wpcom and display them as a list.
+	 * Fetch anonymous data about A12s from wpcom: total count, number of countries, languages spoken.
+	 *
+	 * @since 7.4
+	 *
+	 * @return array $data
+	 */
+	private function fetch_a8c_data() {
+		$data = get_transient( 'a8c_data' );
+		if ( false === $data ) {
+			$data = json_decode(
+				wp_remote_retrieve_body(
+					wp_remote_get( 'https://public-api.wordpress.com/wpcom/v2/jetpack-about' )
+				)
+			);
+			if ( ! empty( $data ) && is_array( $data ) ) {
+				set_transient( 'a8c_data', $data, DAY_IN_SECONDS );
```

Comment:
What do you think about prefixing the transient with Jetpack to keep our data easy to find and / or identify by third-parties?

Reply:
Done